pyvrp/cpp/ProblemData.h

```diff
@@ -47,12 +48,27 @@ class ProblemData
      * @param second Second client.
      * @return distance from the first to the second client.
      */
-    [[nodiscard]] inline int dist(size_t first, size_t second) const;
+    [[nodiscard]] inline distance_type dist(size_t first, size_t second) const;
+
+    /**
+     * Returns the travel duration between the indicated two clients.
+     *
+     * @param first First client.
+     * @param second Second client.
+     * @return duration to travel from the first to the second client.
+     */
+    [[nodiscard]] inline duration_type duration(size_t first,
+                                                size_t second) const;
```

Comment:
Shorten this to `dur`?

Reply:
Could also do `time`, or `travel`? Since we call this a lot I'm looking for something less verbose.

Reply:
I think `duration` is still best as it's clear that it specifies an interval. In that sense, `time` is more ambiguous as it can also refer to a point in `time` (and is used like that in the code). `travel` can mean anything, i.e. `travel_(dist|cost|time)` so I would not use that. If it's that important that it's short, I'd say go with `dur`, it's not immediately clear but also not ambiguous, but I prefer to keep it as is.

Reply:
Works for me. I've mostly kept `duration`, expect one or two places where I needed something slightly shorter and used `dur`.